tests/buyers/helpers/test_ods.py

```diff
@@ -200,15 +196,14 @@ def test_add_font(self):
 
         instance.add_font(fontface)
 
-        instance._document.fontfacedecls.addElement\
-                .assert_called_once_with(fontface)
+        instance._document.fontfacedecls.addElement.assert_called_once_with(fontface)
 
-    def save(self):
+    def test_save(self):
```

Comment:
Again, don't think this is flake8? But good catch.

Or is flake8 smart enough to recognise that this is probably a test case and suggest it has an incorrect name?